app/javascript/src/scenes/bill/components/Content.js

```diff
@@ -1,15 +1,15 @@
 // @flow
 import React, { Component } from 'react'
 import { createFragmentContainer, graphql } from 'react-relay'
-import moment from 'moment'
+import { default as parseDate } from 'date-fns/parse'
```

Comment:
Can these default imports be aliased using the default import syntax?

```
import parseDate from 'date-fns/parse'
```

Reply:
yes, but with generic names like `parse` and `format` I wanted to rename them. I'll put them in our own module and that will help with this.